Catalog/Examples/Combine/Paging/CombinePagingPresenter.swift

```diff
@@ -36,79 +36,38 @@ final class CombinePagingPresenter {
 // MARK: - Extensions -
 
 @available(iOS 13, *)
-extension CombinePagingPresenter: CombinePagingPresenterInterface {
+extension CombinePagingPresenter: CombinePagingPresenterInterface, PageablePresenter {
     func configure(with output: CombinePagingExample.ViewOutput) -> CombinePagingExample.ViewInput {
-        return CombinePagingExample.ViewInput(pokemon: setupPagination(
-            willDisplayLastCell: output.willDisplayLastCell,
-            pullToRefresh: output.restart
-        ))
+        return CombinePagingExample.ViewInput(
+            pokemon:
+                setupPagination(
+                    nextPagePublisher: output.willDisplayLastCell,
+                    reloadPublisher: output.restart,
+                    nextPage: { [unowned self] container, page in
+                        return handleFetchNextData(lastPage: page)
+                    },
+                    hasNextPage: { container, page in
+                        return container.count < (page?.count ?? 0)
+                    }
+                )
+                .map { $0.map { $0 as! Pokemon } }
+                .map({ [unowned self] in createPokemonCellItems(pokemons: $0)})
+                .eraseToAnyPublisher()
```

Comment:
please extract this into a local property and then call it here

Reply:
I don't understand what you mean here, can you please explain